### ISSUE TITLE
Prepend Characters

### ISSUE THREAD

**Issue by jjchawaii** [ThePornDatabase/ThePornDB.bundle#65]
Apologies if this is a dumb question or it's been answered/resolved somewhere already.  I'm trying to set a character to prepend collection names so they appear at the top of the library and are more easily differentiated in the library view.  Namely, trying to use something like the square or dot bullet point.  I've tried this under various fonts, including the plexUI font.  I can paste the character into the space provided and it shows okay, but when going to save, it says "Your changes could not be saved".  Is there a particular font or a particular set of characters that can be used?

On a side note, if I manually go into each collection and rename them, I can use the characters just fine, but any new additions won't go into that collection.  Thanks.

![Screenshot 2024-07-11 214724](https://github.com/user-attachments/assets/528e8ca4-bfe2-47bb-b927-d5915c33005a)

**Comment by DirtyRacer1337** [comment]
just paste and save

**Comment by jjchawaii** [comment]
That’s actually the issue.  Paste is fine, but it will not save if it’s anything other than a character on a standard keyboard.  This may not be an issue, or anything that even be “fixed”.  Just wasn’t sure if I had to use some certain font or any other trick.

TLDR: Paste works / Save does not.

**Comment by DirtyRacer1337** [comment]
I clicked save two time and it's work, anyway not plugin problem

**Comment by jjchawaii** [comment]
Hmm.  Okay, thank you for checking.  I'll tinker with it on my end to find the issue within my setup.  I appreciate your feedback!
